src/content/1.7/modules/sell/techvalidation-checklist.md

```diff
@@ -104,14 +104,39 @@ PrestaShop provides a e-commerce software ready use in many languages. The code
 
 Like for PrestaShop, the code submitted on the marketplace has to be written in English, even if the only user of this code is likely to from only one country or language. A lang unknown by the reviewer would make the validation impossible to do.
 
+#### Risk of conflicts between modules is low
+
+* Configuration keys
+
+Configuration data is shared between the shop and every module installed. This is convenient if your need to get a value from another part of the shop, but include some risks if two modules stores some data in the same key.
+
+Too avoid conflicts, configuration keys must be prefixed by the module name. For instance, using a configuration key in the module `Ps_Checkout` would be:
```

Comment:
```suggestion
Too avoid conflicts, configuration keys must be prefixed by the module name. For instance, using a configuration key in the module `My_Module` would be:
```
Use something more generic maybe? Because people are going to name configuration keys `PS_MY_MODULE` :thinking:

Reply:
Okay, I updated the name for a name without underscore (we aren't supposed to use it)